website/docs/support/product-behaviour.md

```diff
@@ -0,0 +1,9 @@
+# Product behavior
+
+Below is a list of possibly unexpected behaviors/limitations of the project.
+
+## Drive items (OneDrive/SharePoint)
+
+- Link shares when restored always generate different links from the original.
+
+- Anonymous link shares(link shares which aren't associated with any user) aren't restored.
```

Comment:
Any user?  Or any user in the org?  Not sure if there's a technical difference, or just a clarification.
```suggestion
- Anonymous link shares (link shares which aren't associated with any user) aren't restored.
```